caffe2/python/operator_test/detectron_keypoints.py

```diff
@@ -32,8 +32,8 @@ def heatmaps_to_keypoints(maps, rois):
     heights = rois[:, 3] - rois[:, 1]
     widths = np.maximum(widths, 1)
     heights = np.maximum(heights, 1)
-    widths_ceil = np.ceil(widths).astype(np.int)
```

Comment:
is this intentional?

Reply:
Yeah. This is just exporting an internal codemod that did `np.int` -> `int` fbcode-wide since `np.str` is deprecated starting numpy 1.20.0